### PR TITLE
S2 display film info

### PR DESCRIPTION
**Pull request by olivergate**

### CODE REVIEW

File: server/getBondFilms.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//this method will output all 22 bond films
+module.exports.getAllFilms = function (db) {
+    // let collection = db.collection('films')
```
Comment (aabdiaziz) [inline server/getBondFilms.js:3]: extra line not needed

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,49 @@
...
-        </header>
-      </div>
-    );
-  }
-}
+$.get('/films/random').done((res) => {
+    console.log(res)
+    const store = createStore(
+        // reducers,
+        (state = {poo: "shitty poo", action}) => {
```
Comment (aabdiaziz) [inline src/App.js:15]: not needed anymore

Comment (aabdiaziz) [reply src/App.js:15]: s3 changes this to ---> reducers,

File: src/App.js [hunk 3]
```diff
@@ -1,28 +1,49 @@
...
+            return state
+        },
+        {
+            randomFilms: res,
+            title: "abdi"
```
Comment (aabdiaziz) [inline src/App.js]: remove this title.

File: src/App.js [hunk 4]
```diff
@@ -1,28 +1,49 @@
...
+        },
+        applyMiddleware(thunk)
+    )
+
+    const backgroundStyle = {
+        margin: "10px 20px",
+        "background-color": "grey",
```
Comment (aabdiaziz) [inline src/App.js]: take out css and use JS

File: src/App.js [hunk 5]
```diff
@@ -1,28 +1,49 @@
...
+        padding: "30px 90px",
+        "border-radius": "10px",
+        "font-family": "Trebuchet MS"
```
Comment (aabdiaziz) [inline src/App.js]: same as above for any style key in quotes ""

File: src/DisplayFilmInfo.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+
+class DisplayFilmInfo extends Component
```
Comment (aabdiaziz) [inline src/DisplayFilmInfo.js:3]: depricated code not need for this project anymore.

File: src/components/Films.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    margin: "0 auto"
+}
+
+export default ({films}) => (
+    <div style={styleFilmSection}>
+
+        {films.map((film) => {
+            return <Film title ={film.name} imgUrl={film.image} date={film.date}/>
+        })}
+
```
Comment (aabdiaziz) [inline src/components/Films.js]: empty line breaks. take them out pls

**Review by jaholt — DISMISSED** [review]

**Review by Skylucas — COMMENTED** [review]
Date needs to be shown

File: server/server.js [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+const BondFilms = require('./getBondFilms')
+
+const app = express()
+const port = 3001
+
+//connecting to NOSQL Mongo -> connection to the URL
+const url = 'mongodb://localhost:27017'
+//Database Name
+const dbName = 'james_bond'
+//instantiating new mongo client
```
Comment (robertkieronreeves) [inline server/server.js:14]: As above.

File: src/App.js [hunk 9]
```diff
@@ -1,28 +1,48 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import { render } from 'react-dom'
+// import App from './components/App'
```
Comment (robertkieronreeves) [inline src/App.js:3]: Delete line if not required.

File: src/App.js [hunk 10]
```diff
@@ -1,28 +1,48 @@
...
+import { Provider } from 'react-redux'
+import { createStore, applyMiddleware } from 'redux'
+// import reducers from './reducers'
```
Comment (robertkieronreeves) [inline src/App.js:6]: Delete if not required.

File: src/App.js (hunk 3 above)
Comment (aabdiaziz) [reply src/App.js]: removed

File: src/App.js (hunk 4 above)
Comment (aabdiaziz) [reply src/App.js]: done

File: src/App.js (hunk 5 above)
Comment (aabdiaziz) [reply src/App.js]: done

File: src/components/Films.js (hunk 7 above)
Comment (aabdiaziz) [reply src/components/Films.js]: done

**Review by Skylucas — APPROVED** [review]

File: src/components/Film.js [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+const button = {
+    width: "100%",
+    height: "50px"
+}
+
+export default ({title,imgUrl,date}) => (
+    <div style={styleFilms}>
+        <h3 style={filmTitle}>{title}</h3>
```
Comment (robertkieronreeves) [inline src/components/Film.js:33]: Perhaps a line break between the title and the Bond poster?

**Review by jaholt — APPROVED** [review]

File: src/App.js (hunk 9 above)
Comment (aabdiaziz) [reply src/App.js:3]: done

File: src/App.js (hunk 10 above)
Comment (aabdiaziz) [reply src/App.js:6]: done

File: src/components/Film.js (hunk 11 above)
Comment (aabdiaziz) [reply src/components/Film.js:33]: done

File: server/server.js (hunk 8 above)
Comment (aabdiaziz) [reply server/server.js:14]: instruction not code

**Pull request merged**